php/Terminus/Collections/UserMemberships.php

```diff
@@ -0,0 +1,108 @@
+<?php
+
+namespace Terminus\Collections;
+use Terminus\Request;
+use Terminus\Session;
+use Terminus\UserMembership;
+use \Terminus_Command;
+
+class UserMemberships {
```

Comment:
Isn't OrganizationUserMemberships a misnomer? Organizations aren't users, are they? I figured we'd use UserMemberships and OrganizationMemberships.